internal/framework/runnables/cronjob_test.go

```diff
@@ -44,3 +48,38 @@ func TestCronJob(t *testing.T) {
 	g.Eventually(errCh).Should(Receive(BeNil()))
 	g.Eventually(errCh).Should(BeClosed())
 }
+
+func TestCronJob_ContextCanceled(t *testing.T) {
+	g := NewWithT(t)
+
+	readyChannel := make(chan struct{})
+
+	timeout := 10 * time.Second
```

Comment:
Since we are only testing cancelation here, we can remove lines 57-64 and just pass `func(context.Context){}` as the worker in the cron job config